### DISCUSSION TITLE
New Version for iOS 10 support?

### DISCUSSION THREAD

**Discussion by CTaylorOTG** [software-mansion/react-native-screens#1497] [Q&A] [answered]
When is the next version going to be released? and if there is no plan would it be possible to get a small release to get the fix in commit https://github.com/software-mansion/react-native-screens/pull/1423 out? Currently in 1.13.1 RNScreens is causing a crash on iOS 10 because that issue.

**Comment by tboba** [comment, marked as answer]
Hi @CTaylorOTG, the newest react-native-screens version (3.26.0) should already contain your contribution 🎉 🎉 

Cheers!
